Gruntfile.js

```diff
@@ -257,6 +257,15 @@ module.exports = function(grunt) {
         ],
       },
 
+      // Insert mustachio syntax on Postmark layout preheaders
+      postmarkPreheader: {
+        src: [`${path.dist}postmark-layouts/**/content.hbs`],
+        overwrite: true,
+        replacements: [
+          { from: /({{)(.?pm:preheader.?)(}})/g, to: '\\{{#preheader}}<span class="preheader">\\{{.}}</span>\\{{/preheader}}' },
```

Comment:
Replaces the preheader tag in the handlebars template to Postmark-compatible Mustachio syntax.

`{{pm:preheader}}`
to
`{{#preheader}}<span class="preheader">{{.}}</span>{{/preheader}}`

Reply:
This requires that the `preheader` text is included inside of the template's model since we can't insert the text from the template HTML.

Reply:
@derekrushforth would you mind clarifying what "This requires that the preheader text is included inside of the template's model" means please?

Reply:
Nevermind, I saw your comment in #46

Reply:
@philefstat 👍 Merging this to master soon.